Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -1268,6 +1268,9 @@ static Ref<CSSValue> valueForScrollbarGutter(const ScrollbarGutter& gutter)
 
 static Ref<CSSValueList> valueForTextBoxEdge(const TextBoxEdge& textBoxEdge)
 {
+    if (textBoxEdge.over == TextBoxEdgeType::Leading && textBoxEdge.under == TextBoxEdgeType::Leading)
+        return CSSValueList::createSpaceSeparated(createConvertingToCSSValueID(textBoxEdge.over));
```

Comment:
```suggestion
    if (textBoxEdge.over == TextBoxEdgeType::Leading && textBoxEdge.under == TextBoxEdgeType::Leading)
        return createConvertingToCSSValueID(textBoxEdge.over);
```

This should also work, and you change the return type  of the function to `Ref<CSSValue>`.

Reply:
fixed :)